src/app/components/client/exposure_card/ExposureCard.test.tsx

```diff
@@ -85,6 +86,21 @@ describe("ScanResultCard", () => {
     expect(innerDescription).toBeInTheDocument();
   });
 
+  // Data broker removal under maintenance
+  it("shows the right description for a scan result card where removal is under maintenance", () => {
```

Comment:
Nit: Should we also add a test that checks that the description is _not_ shown for other statuses?